internal/template/template.go

```diff
@@ -0,0 +1,118 @@
+// Package template provides utility functions around reading and writing templates.
```

Comment:
Time to add license to all files :P at least all the go files.

Reply:
Done. I always forget :(